### ISSUE TITLE
Double scrollbar when overflow-x:hidden added to html and body

### ISSUE THREAD

**Issue by Ghost180195** [jlmakes/scrollreveal#385]
Hello again!
OS version: Windows 10
Browser version: This issue appears in all browsers.
ScrollReveal version: 3.3.6
Codepen link: https://codepen.io/anon/pen/jGXzOJ
When I scroll to bottom the second scrollbar disappears. If in page we don't have animations that appears from bottom then everything is fine.

**Comment by jlmakes** [comment]
Hey @Ghost180195, two things:

1. As brought up in #379, `overflow` should not be applied to `<html>`, as it can cause a serious issue with determining element visibility. To be honest, I'm not actually sure why hiding horizontal overflow causes a vertical scrollbar, but removing `overflow` also removes the extra scrollbar.

2. The other thing to be aware of is that you're using:

```js
var fromBottom = {
  origin   : "bottom",
  distance : "80px",
}
```

If you use this in an element that's near the bottom of your `<body>`, the pre-reveal styles will push the element _outside_ of the bottom border:

![screen shot 2017-10-18 at 1 07 15 pm](https://user-images.githubusercontent.com/2044842/31715774-5fc105b4-b405-11e7-96bd-a40fcf8bd569.png)

So when the last element reveals, it comes back fully within the `<body>`, removing the overflow. In my testing, this normally causes a small "jump" as the body height shifts... but for you it seems, when using `overflow-x` on the `<html>`, you get an an extra scrollbar.

***

So basically, you should remove the `overflow-x` from `<html>` as described in **(1)** to fix your extra scrollbar (and prevent bugs), and be mindful of the browser's natural overflow behavior described in **(2)** when you're using reveals that push elements outside the `<body>` (in any direction).

**Comment by Ghost180195** [comment]
Thanks for your answer!
But if I remove `overflow-x` from` <html>` then responsive design will be broken (look at this link: https://github.com/jlmakes/scrollreveal/issues/377)

**Comment by jlmakes** [comment]
I recognize that `overflow-x` on `<html>` solved #377 for you, but I am no longer recommending anyone do that due to changes in WebKit (Chrome 61) as detailed in #379.

The way I see it (based on your Codepen demonstration), your problem is the result of **(1)** using `overflow-x` on `<html>` and **(2)** using reveal options that move your elements off the bottom of your page; you can fix the extra scrollbar by addressing either factor.
